api/models.py

```diff
@@ -68,30 +69,54 @@ def __str__(self):
 
 class Item(models.Model):
     """Stores all items"""
+
+    status_list = ["Available", "Allocated", "Lost", "Damaged"]
+
+    item_statuses = (
+        (status_list[0], "Available"),
+        (status_list[1], "Allocated"),
+        (status_list[2], "Lost"),
+        (status_list[3], "Damaged")
+    )
```

Comment:
from [here](https://docs.djangoproject.com/en/2.0/ref/models/fields/#django.db.models.Field.choices) -> "Generally, it’s best to define choices inside a model class, and to define a suitably-named constant for each value".. i think that'd be cleaner.